recipes/pelicanplatform/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pelicanplatform" %}
+{% set version = "7.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PelicanPlatform/pelican/archive/v{{ version }}.tar.gz
+  sha256: 6561a188965d2d366a07272c59d0aee9e41db27f3eab04fc3ae8a1760b887b63
```

Comment:
Add https://gitlab.com/cznic/mathutil/-/blob/master/LICENSE?ref_type=heads as another source and list it under licenses. Then the recipe 👍

Reply:
Thanks @xhochy, this has been added.